Atualização de versões do php e mysql

GabrielDully opened the pull request:
Fix #1

GabrielDully committed:
Atualização de .lando.yml:

 - php:7.4 -> php:8.2

 - mysql:5.7 -> mysql:8.0

GabrielDully committed:
Atualização de Dockerfile.custom:

 - php:7.4-apache-2 -> php:8.2-apache-4

 - remocao do parâmetro oci8-2.2.0 na instalação do Microsoft SQL Server

alecostaweb commented:
Eu prefiro com espaço, mas neste caso tanto faz.

leandroembu commented:
@GabrielDully sobre o estilo do comentário, tanto faz. Se eu for editar um documento de outra pessoa, vou seguir o estilo dela. Se eu for iniciar um novo documento, uso meu estilo (que é com o espaço).

GabrielDully committed:
Fix #1 - Atualização de .lando.yml e Dockerfile.custom

 - Reimplementação das linhas deletadas em comentários para versões com versões anteriores do php e mysql.

GabrielDully commented:
Obrigado pelos feedbacks, pessoal!

Já atualizei a minha Fork. Assim, acredito que já seja possível fazer o Merge, se for o caso.

GabrielDully commented:
> Adiciona no readme que atualmente o container será com php8.2

GabrielDully committed:
Alteração do README.md:

 - Inserção da nota referente às versões do PHP e MySQL utilizadas no container.
